skilloracle/endpoint/skilloracle.py

```diff
@@ -9,41 +9,54 @@
 except:
     from ..skilloracle import SkillOracle
 
-
-# This endpoint is WIP
 class SkillOracleEndpoint(object):
     def __init__(self, fetcher=None):
-        self.oracle = SkillOracle(port=7000)
+        self.host = "skilloracle"
+        self.oracle = SkillOracle(host=self.host, port=7000)
         self.put_valid_keys = {'label', 'name', 'context'}
         self.fetcher = fetcher
         if not fetcher:
             fetcher = None # what kind of default woudl we do here?
 
     def on_put(self, req, resp):
         query = falcon.uri.parse_query_string(req.query_string)
+        # ^ just use req.params.items or, below, req.params.keys()
         query_keys = set(query.keys())
 
         if self.put_valid_keys.issuperset(query_keys):
-            label = query['label']
-            name = query['name']
-            context = query['context']
+            print(req.params)
```

Comment:
Whether or not we do want to log this particular line, I think we should figure out some kind of logging solution. It's been a little while since I fiddled around with logging in a Docker container, but it handles this quasi-automatically, right? If we make `logging.info` it will show up in the docker `log` command for the container?

Reply:
That is basically correct, I believe you can just print to stdout out and your output will be available under `docker logs <service name>`. 

Is there a particular logging format you recommend that we follow? I can add a print statement that follows that format so everything is viewable with `docker logs <service name>`

Reply:
Will leave logging of the request, request parameters as is for this PR, will improve later.